.gitignore

```
@@ -72,5 +72,14 @@ iree/builtins/**/bin/*.ll
 *.tar
 *.tar.*
 
+# VS Code DevContainer
+.devcontainer/*
+!.devcontainer/README.md
+!.devcontainer/configure.py
+!.devcontainer/devcontainer.json
+!.devcontainer/docker-compose.base.yml
+!.devcontainer/docs_data/
+!.devcontainer/docs_data/*.png
```

Are there any other files that get put in devcontainer? Or do we expect users to put things there? Seems like it would be easier to just gitignore `.devcontainer/docker-compose.yml`

```suggestion
.devcontainer/docker-compose.yml
```

---

+1 to applying the suggested edit here. We were talking about explicit gitignore changes in the context of having a file checked in that developers could make edits to without git picking them up. Now that the `docker-compose.yml` file is only generated from the configure script, we can just gitignore _that one file_ directly.

---

Works for me